userspace/libsinsp/filter/ast.h

```diff
@@ -117,9 +145,15 @@ struct SINSP_PUBLIC string_visitor: public expr_visitor
 */
 struct SINSP_PUBLIC expr
 {
+    expr();
+    expr(const pos_info& pos);
     virtual ~expr() = default;
     virtual void accept(expr_visitor*) = 0;
     virtual bool is_equal(const expr* other) const = 0;
+
+    const pos_info& get_pos();
```

Comment:
```suggestion
    const pos_info& get_pos() const;
```
If you decide to keep it.

Reply:
Sure, added.